okhttp/src/main/java/okhttp3/internal/Platform.java

```diff
@@ -176,6 +193,13 @@ private static Platform findPlatform() {
     return new Platform();
   }
 
+  /**
+   * Is Java 9.  Does not include Java 10 or later.
```

Comment:
In this case I think you wanna do the method lookup that you have in `Jdk9Platform` and ignore failures in the catch clause if that misses